Assets/Mirror/Runtime/NetworkManagerHUD.cs

```diff
@@ -74,6 +103,44 @@ void StartButtons()
         {
             if (!NetworkClient.active)
             {
+            
+                GUILayout.BeginHorizontal();
+                {
+                    GUILayout.Label($"IP Type : {_ipType[_ipTypeSelectedIndex]}");
+                    {
+                        if (GUILayout.Button("▼"))
+                        {
+                            _ShowIpTypeDropdown = !_ShowIpTypeDropdown;
+                        }
+                    }
+                }
+                GUILayout.EndHorizontal();
+
+                if (_ShowIpTypeDropdown)
+                {
+                    using (GUILayout.ScrollViewScope scope = new GUILayout.ScrollViewScope(scrollViewVector))
+                    {
+                        scrollViewVector = scope.scrollPosition;
+                        for (int index = 0; index < _ipType.Length; index++)
+                        {
+                            if (!GUILayout.Button(_ipType[index]))
+                            {
+                                continue;
+                            }
+                            else
+                            {
+                                if (index == 0) { _IPType = IPType.localhost; }
```

Comment:
as mentioned above, please simply show local ip and public ip. not so much magic